.gitlab/llnl-ci-spack.yml

```diff
@@ -55,10 +52,9 @@ stages:
     - spack config add concretizer:unify:true
     - spack config add concretizer:reuse:false
     - spack config add config:db_lock_timeout:300
-    - spack compiler find
-    - spack add ${COMPILER} ${TARGET_ARCH_OPT}
+    - module --latest load gcc && spack compiler find
+    - spack add rust ^openssl certs=system openjdk ^openssl certs=system
```

Comment:
let's remove `openjdk ^openssl certs=system` and resolve OpenJDK issues outside of this PR

Reply:
Done